webapp/WEB-INF/templates/admin/plugins/forms/multiview/panel/multiview_form_panel_template.html

```diff
@@ -1 +1,3 @@
-<@tabLink active=panel_active href='jsp/admin/plugins/forms/MultiviewForms.jsp?plugin_name=forms&selected_panel=${panel_technical_code}&change_panel=true' title='${panel_title}'><@tag color='primary' class='ml-1 ms-1'>${panel_formResponseNumber}</@tag></@tabLink>
\ No newline at end of file
+<#assign hrefValue>jsp/admin/plugins/forms/MultiviewForms.jsp?plugin_name=forms&selected_panel=${panel_technical_code}&change_panel=true</#assign>
```

Comment:
The goal of this commit is to add a "id_form" parameter.
... but how does it solve the problem of displaying the status filter ?

Reply:
When we clic on the link, a call is made to _MultiviewFormsJspBean.getMultiviewFormsView_ through which we build the various filters including the status/state filter.
The class responsible of building the tempate of this filter is FormFilterDisplayWorkflowState (method buildTemplate) where the **id_form** is retrived from the request submitted earlier.
Thanks to the id_form, the "referenceList" variable is populated and then the template is build with this information.